format.go

```diff
@@ -60,6 +60,10 @@ func toPprof(s map[string]int, hz int) *profile.Profile {
 			Unit: "nanoseconds",
 		},
 	}
+	p.PeriodType = &profile.ValueType{
+		Type: "cpu",
```

Comment:
I think this is misleading. fgprof does not produce CPU profiles. It produces "wallclock" profiles.

So maybe this?

```suggestion
		Type: "wallclock",
```

Reply:
Fixed.